src/components/Tabs/Annotation/AnnotationImportButton.tsx

```diff
@@ -134,7 +142,35 @@ export default function AnnotationImportButton(props: AnnotationImportButtonProp
             </Upload.Dragger>
           )}
           {parseResult && hasExistingAnnotationData && (
-            <MessageCard type="warning">Existing annotations will be overwritten during import.</MessageCard>
+            <MessageCard type="info">
+              <FlexColumn>
+                <span style={{ marginBottom: 5 }}>
+                  You have existing annotations. How would you like to handle the imported annotations?
+                </span>
+                <Radio.Group value={mergeMode} onChange={(e) => setMergeMode(e.target.value)}>
+                  <Space direction="vertical">
+                    <Radio value={AnnotationMergeMode.APPEND}>Append as new annotations</Radio>
+                    <MultilineRadio
+                      value={AnnotationMergeMode.MERGE}
+                      $expanded={mergeMode === AnnotationMergeMode.MERGE}
+                    >
+                      {mergeMode === AnnotationMergeMode.MERGE ? (
+                        <FlexColumn>
+                          <p style={{ margin: 0 }}>Merge matching annotations</p>
+                          <p style={{ margin: 0 }}>
+                            Matching labels will be merged, all other labels will be appended. If there are per-object
```

Comment:
What is a per-object conflict? Is this if the same object has the same label in both annotation sets, but different values?

Reply:
Yes, exactly. I'll have to think about a better wording on this one... maybe "If there are conflicts where the same object is annotated with different values..."